docs/auth/README.md

```diff
@@ -81,16 +79,20 @@ curl -H "Authorization: Bearer $API_TOKEN"
 </template>
 </CodeSwitcher>
 
+::: warning
+Keep in mind at this point in time only JS SDK is automatically refresh access token. If you use REST API or Python SDK you need to handle this situation by your own. Token live time 2h 
+:::
+
 ## Project Users
-Another way to authenticate within Jexia is to use **Project Users**. Usually, you need this when you want to provide more rights to specific users. Let's say **Update** and **Delete** actions for your blog. Firstly, you need to create a user under the **Project Users** section.
+Another way to authenticate within Jexia is to use **Project Users**. Usually, you need this when you want to provide more rights to specific users. Let's say **Update** and **Delete** actions for your blog can be done by owners only. Firstly, you need to create a user under the **Project Users** section.
 
 ![UMS Users](./ums-2.png)
 
-After this, you need go to the **Policies** section and create a new policy, ensuring that **Subject** is selected as **AllUsers** and the selected Datasets within **Resource** are what you want these users to be able to access. Finally, sselect all **Actions** you want these users to perform.
+After this, you need go to the **Policies** section and create a new policy, ensuring that **Subject** is selected for some specific **Namespace** or **User account** and **Resource** are selected to specific dataset, fileset or AllUsers(for user table) what you want these users be able to run actions. Finally, select all **Actions** you wanted these users to perform.
 
 ![Policy](./policy.png)
 
-This will allow any registered user to have full CRUD access to the example `orders` dataset. 
+This will allow any user from namespace `registered` or user with email `admin@x.com` to have Update & Delete operations for all own records and Read operation for all records which has schema field `confirmed` equal to `true`
```

Comment:
```suggestion
This will allow any user from the namespace `registered` or the user with the email `admin@x.com` to have **Update & Delete** operations for all their records and the **Read** operation for all the records which have a schema field `confirmed` equal to `true`.
```